javascript/javascript3/week2/visual-promise/main.js

```diff
@@ -0,0 +1,13 @@
+async function translateOneByOne() {
+    await moveElement(document.getElementById("red-circle"), { x: 20, y: 300 });
+    await moveElement(document.getElementById("blue-circle"), { x: 400, y: 300 });
+    await moveElement(document.getElementById("green-circle"), { x: 400, y: 20 });
+}
+
+function translateAllAtOnce() {
+    Promise.all(
+        moveElement(document.getElementById("red-circle"), { x: 20, y: 300 }),
+        moveElement(document.getElementById("blue-circle"), { x: 400, y: 300 }),
+        moveElement(document.getElementById("green-circle"), { x: 400, y: 20 })
+    )
```

Comment:
The `Promise.all()` method takes an array of promises as an input an returns a promise that resolves to an array of the results of the input promises. So you can pass `Promise.all()` an array of promises something like this:

```suggestion
    Promise.all([
        moveElement(document.getElementById("red-circle"), { x: 20, y: 300 }),
        moveElement(document.getElementById("blue-circle"), { x: 400, y: 300 }),
        moveElement(document.getElementById("green-circle"), { x: 400, y: 20 })
    ])
```

And then you can use `.then(()=> console.log("All elements have been moved"))` to log out something after all elements have been moved

Reply:
Thank you for review! I`ve updated my pr